recipe/meta.yaml

```diff
@@ -11,19 +11,19 @@ source:
   patches:
     # https://github.com/abetlen/llama-cpp-python/commit/e3941d9c674dbd9891dc3ceda390daeb21f05fd1
     - e3941d9.patch 
+    - osx-64-pick-discrete.patch  # [osx and x86_64]
```

Comment:
@YYYasin19, the GPU detection does not work correctly in MacOS Intel.
this is a patch to fix the issue.